poulpe-service/src/test/java/org/jtalks/poulpe/service/transactional/TransactionalUserServiceTest.java

```diff
@@ -411,11 +396,15 @@ private PoulpeUser createPoulpeUserWithPredefinedNameAndGroups(List<Group> group
         return user;
     }
 
-
     public static PoulpeUser user() {
         return new PoulpeUser(RandomStringUtils.randomAlphanumeric(10), "USERNAME@mail.com", "PASSWORD", "salt");
     }
 
+    private PoulpeUser makePoulpeUser(String username, String password) {
```

Comment:
There is a method above - user(). You can overload it and have `user(username, password)` - will look nicer.